src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas2/Cas2ApplicationTest.kt

```diff
@@ -179,6 +183,112 @@ class Cas2ApplicationTest : IntegrationTestBase() {
   @Nested
   inner class GetToIndex {
 
+    @Test
```

Comment:
could we delete the `Get all applications returns 200 with correct body` now as a I guess the same functionality will be tested by this test? aware that this file is getting really long now!